src/components/SelectionList/BaseListItem.tsx

```diff
@@ -4,34 +4,31 @@ import Icon from '@components/Icon';
 import * as Expensicons from '@components/Icon/Expensicons';
 import OfflineWithFeedback from '@components/OfflineWithFeedback';
 import PressableWithFeedback from '@components/Pressable/PressableWithFeedback';
-import Text from '@components/Text';
-import useLocalize from '@hooks/useLocalize';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 import CONST from '@src/CONST';
-import RadioListItem from './RadioListItem';
 import type {BaseListItemProps, ListItem} from './types';
-import UserListItem from './UserListItem';
 
 function BaseListItem<TItem extends ListItem>({
     item,
-    isFocused = false,
+    wrapperStyle,
+    selectMultipleStyle,
```

Comment:
any need for this? we can just use the style directly in here.

Reply:
no, we need to be able to style it from the outside - that's the whole point of this refactor, so we definitely need this